dask/base.py

```diff
@@ -633,7 +633,6 @@ def normalize_dataframe(df):
 def register_numpy():
     import numpy as np
 
-    @normalize_token.register(np.ndarray)
```

Comment:
Please re-add this line, and remove the added line at 664. This change is unnecessary.

Reply:
I wasn't sure if the decorators had some side effects; as I'm using `normalize_array` inside `normalize_matrix`..  Reverted it following your comment.  Thanks for the review!